pandas/tseries/resample.py

```diff
@@ -14,7 +21,560 @@
 import pandas.tslib as tslib
 
 
-_DEFAULT_METHOD = 'mean'
+class Resampler(_GroupBy):
+
+    # to the groupby descriptor
+    _attributes = ['freq', 'axis', 'closed', 'label', 'convention',
+                   'loffset', 'base', 'kind']
+
+    # API compat of allowed attributes
+    _deprecated_valids = _attributes + ['_ipython_display_', '__doc__',
+                                        '_cache', '_attributes', 'binner',
+                                        'grouper', 'groupby', 'keys',
+                                        'sort', 'kind', 'squeeze',
+                                        'group_keys', 'as_index',
+                                        'exclusions']
+
+    # API compat of disallowed attributes
+    _deprecated_invalids = ['iloc', 'loc', 'ix']
```

Comment:
I have to do this, because I don't know when they access the attribute if its a get or a set. (and actually trying to figure this out would require me to really hack the internals)

e.g.
``r.ix[...]`` looks the same as ``r.ix[...] = 5``